app/lib/preservation_catalog/s3_audit.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module PreservationCatalog
+  # Base class for AWS and IBM audit classes
+  class S3Audit
+    attr_reader :zmv, :results
+
+    # @param [ZippedMoabVersion] the ZippedMoabVersion to check
+    # @param [AuditResults] the AuditResults instance used to track findings for this audit run
+    def initialize(zmv, results)
+      @zmv = zmv
+      @results = results
+    end
+
+    # convenience method for instantiating the audit class and running the check in one call
+    def self.check_replicated_zipped_moab_version(zmv, results)
+      new(zmv, results).check_aws_replicated_zipped_moab_version
+    end
+
+    def check_aws_replicated_zipped_moab_version
```

Comment:
If this is given a generic name, then `self.check_replicated_zipped_moab_version` could move to the superclass.

Reply:
i think it's already in the superclass?

but agreed, the name could be genericized, just looked closely by playing around with it in a text editor, and other than the method names and some minor spacing and var name differences, `check_aws_replicated_zipped_moab_version` is exactly the same (in terms of functionality) as `check_ibm_replicated_zipped_moab_version`.

Reply:
so i think this method is essentially always being called, and i think maybe `check_ibm_replicated_zipped_moab_version` isn't being called at all, but it doesn't matter, because both do the exact same thing.  that might also explain the coverage drop?  `check_ibm_replicated_zipped_moab_version` is essentially no longer being exercised?